flytepropeller/pkg/controller/nodes/attr_path_resolver.go

```diff
@@ -11,14 +12,16 @@ import (
 	"github.com/flyteorg/flyte/flytestdlib/storage"
 )
 
+const messagepack = "msgpack"
```

Comment:
Please move this const to one place and reference it everywhere...

Reply:
I'll move it now.